packages/app/src/app.ts

```diff
@@ -87,64 +91,32 @@ export class App<Req extends Request = Request, Res extends Response = Response>
     this.cache = {}
   }
 
-  /**
-   * Set app setting
-   * @param setting setting name
-   * @param value setting value
-   */
   set<K extends keyof AppSettings>(setting: K, value: AppSettings[K]): this {
```

Comment:
i suppose jsdoc comments get inherited from the interface, is that correct?

Reply:
Definitely. We could take the description of the class (line 38)  into the interface too, but I don’t know if it would be ok..

Reply:
Jsdoc for set added to interface, thanks